README.md

```diff
@@ -160,6 +160,37 @@ ansible-playbook ansible/provision-tls-cluster.yml \
 --private-key '<path to a private key with ssh access to the hosts>'
 ```
 
+## Building a Cluster with Tiered Storage
+
+To enable Tiered Storage on a cluster you need the following two values:
+
+```yaml
+cloud_storage_region: "<<your bucket region>>"
+tiered_storage_bucket_name: "<<your bucket name>>"
+```
+
+You also need a method of authenticating to the bucket. You can use either credentials in the instance
+
+```yaml
+cloud_storage_credentials_source: "AWS instance metadata"
```

Comment:
[question] The doc says that you need:

```
cloud_storage_region: "<<your bucket region>>"
tiered_storage_bucket_name: "<<your bucket name>>"
```

PLUS either:
```
cloud_storage_credentials_source: "AWS instance metadata"
```

or credentials passed into the playbook:
```
--extra-vars cloud_storage_access_key="<access_key>" \
--extra-vars cloud_storage_secret_key="<secret_key>"
```

and in the following example (L183) we see:

```
ansible-playbook ansible/playbooks/provision-tls-cluster.yml \
-i hosts.ini \
--private-key '<path to a private key with ssh access to the hosts>' \
--extra-vars create_demo_certs=false \
--extra-vars advertise_public_ips=false \ 
--extra-vars handle_certs=false \
--extra-vars redpanda_truststore_file='<path to ca.crt file>' \
--extra-vars cloud_storage_access_key="<access_key>" \
--extra-vars cloud_storage_secret_key="<secret_key>"
```

And I don't see `cloud_storage_region` nor `tiered_storage_bucket_name` so I'm confused about how to set these values.

Can you provide an example for each one of them?